development/etiquette.md

```diff
@@ -0,0 +1,49 @@
+# Etiquette
+> Karen has an etiquette? Strange 🤔
+
+Listen up everyone, this will address the Karen Bot etiquette
+
+## Commit etiquette
+- Only push small commits (e.g if you changed 2 files, commit one file, commit the other and only <i>then</i> push) 🤏
+- Keep commit titles short and explain them in the description of the commit 
+- **DO NOT PUSH TO MASTER BRANCH!** Push to a new branch (e.g username-whatyouchanged and create a PR)
```

Comment:
Don't be racist lady, change that master wording for main, since your branch is also correctly named. Also in the main readme of Karen repository now that I remember.

Reply:
Æ fine

Reply:
Love the current PR etiquette. Does the `whatyouchanged` part include a suggested change quantifier (major change, minor change, patch)?